src/test/java/org/skriptlang/skript/test/tests/localization/UtilsPlurals.java

```diff
@@ -0,0 +1,46 @@
+package org.skriptlang.skript.test.tests.localization;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import ch.njol.skript.util.Utils;
+
+public class UtilsPlurals {
+
+	/**
+	 * Testing method Utils.getEnglishPlural
```

Comment:
This can be a link `{@link Utils#toEnglishPlural(String)}`

Reply:
It's a testing class and the javadoc won't be seen, but ok, applied.